articles/faq/increase-limit-on-existing-paid-plan.md

```diff
@@ -0,0 +1,6 @@
+---
+ title: "If I stay on my plan, can I increase my daily limits?"
```

Comment:
Need to remove space in front of `title` and `description` - this metadata is YAML and the syntax is strict. If in doubt, you should always click the "view" button in github to see if things parse properly.